homeassistant/components/zwave_js/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "Z-Wave JS",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/zwave_js",
-  "requirements": ["zwave-js-server-python==0.9.0"],
+  "requirements": ["zwave-js-server-python==0.10.0"],
```

Comment:
The bump is merged in dev so we should rebase here so this change isn't included here.